iroh-net/src/magicsock.rs

```diff
@@ -1638,34 +1587,6 @@ enum DiscoBoxError {
 
 type RelayRecvResult = Result<(PublicKey, quinn_udp::RecvMeta, Bytes), io::Error>;
 
-/// Reports whether x and y represent the same set of endpoints. The order doesn't matter.
```

Comment:
This code is removed.  This custom eq was needed to make the Watchable behave correctly, but also caused the bug that the updated_at field of the DiscoveredDirectAddrs was never updated.